apps/web/app/providers.tsx

```diff
@@ -0,0 +1,15 @@
+'use client'
+
+import posthog from 'posthog-js'
+import { PostHogProvider as PHProvider } from 'posthog-js/react'
+
+if (typeof window !== 'undefined') {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY as string, {
+    // api_host: process.env.NEXT_PUBLIC_POSTHOG_HOST as string,
```

Comment:
I'm finding Next.js proxies to be flakey when using rewrites, and I can't seem to get the middleware to hit. Since we don't have the domain in Cloudflare, we can't use a CF worker proxy. @steven-tey maybe you can help me get the middleware to hit and we can follow these https://posthog.com/docs/advanced/proxy/nextjs-middleware